cmd/prometheus/main.go

```diff
@@ -186,6 +202,12 @@ func reloadConfig(filename string, rls ...Reloadable) bool {
 	for _, rl := range rls {
 		success = success && rl.ApplyConfig(conf)
 	}
+	if success {
+		configSuccess.Set(1.0)
+		configSuccessTime.Set(float64(time.Now().Unix()))
+	} else {
+		configSuccess.Set(0)
```

Comment:
(and then in line 200, `success := true` would have to be changed to `success = true` to not shadow the named return variable)